src/test/java/com/wikia/webdriver/testcases/discussions/PostsList.java

```diff
@@ -75,4 +122,30 @@ public void userCanSortPostsList() {
     Assertion.assertTrue(postsList.clickOnSortButtonMobile().isSortListVisibleMobile());
     Assertion.assertEquals(postsList.clickOnTrendingLinkMobile().getSortButtonLabel(), "Trending");
   }
+
+  public void userCanViewMorePosts() {
+    PostsListPage postsList = new PostsListPage(driver).open();
+    int startingListLength = postsList.getPostsListLength();
```

Comment:
no need to store value in variable, you can use `postsList.getPostsListLength()` in Assertion directly